docs/releasenotes/5.4.0.rst

```diff
@@ -1,9 +1,29 @@
-5.4.0 (unreleased)
+5.4.0
 -----
 
 API Changes
 ===========
 
+APNG extension to PNG plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Animated Portable Network Graphics (APNG) images are not fully supported but
+can be opened via the PNG plugin to get some basic info::
+
+    im = Image.open("image.apng")
+    print(im.mode)  # "RGBA"
+    print(im.size)  # (245, 245)
+    im.show()  # Shows a single frame
+
+Check for libjpeg-turbo
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can check if Pillow has been built against the libjpeg-turbo version of the
+libjpeg library::
+
+    from PIL import features
+    features.check_feature("libjpeg_turbo")  # True or False
```

Comment:
Maybe you feel like this is an edge case because it would require Pillow to not be installed correctly, but [`None` is also a possible return value](https://github.com/python-pillow/Pillow/blob/master/src/PIL/features.py#L69)

Reply:
Yeah, this is just an example and not thorough docs. It could also raise `ValueError`. Thanks though!